src-electron/db/zap-schema.sql

```diff
@@ -808,6 +808,7 @@ CREATE TABLE IF NOT EXISTS "ENDPOINT" (
   "PROFILE" integer,
   "ENDPOINT_IDENTIFIER" integer,
   "NETWORK_IDENTIFIER" integer,
+  "PARENT_ENDPOINT_REF" integer NULL,
```

Comment:
You need a foreign key constraint here. It needs to be a foreign key into ENDPOINT_ID. 'on delete set nul' feels right.

Reply:
May be an on delete cascade? You may be right though. We probably do not want a user to loose their endpoint altogether.

Reply:
okay, thanks, will do